pkg/kubectl/cmd/get.go

```diff
@@ -182,13 +183,22 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 		return cmdutil.UsageError(cmd, usageString)
 	}
 
-	// always show resources when getting by name or filename
+	// determine if args contains "all"
+	for _, a := range args {
+		if a == "all" {
```

Comment:
@pwittrock thanks for the feedback. I removed this and just use the boolean value of `resource.MultipleTypesRequested`.

cc @fabianofranz this would mean that passing the `all` alias *or* requesting mixed resource types would cause the value of `show-all` to default to `true`. wdyt?